client/src/main/java/tds/assessment/Assessment.java

```diff
@@ -14,6 +16,7 @@
     private String subject;
     private List<ItemConstraint> itemConstraints = new ArrayList<>();
     private List<Segment> segments = new ArrayList<>();
+    private Set<Strand> strands = new HashSet<>();
```

Comment:
I know I did this first but I'm wondering if instead of setting sets and lists here if instead we should tweak the getters.

```
Set<Strand> getStrands() {
  return strands != null ? strands : new HashSet<>();
}